### PR TITLE
feat: union transfer solidity example

### PR DESCRIPTION
**Pull request by o-az**

### CODE REVIEW

File: docs/src/content/docs/integrations/ucs01/solidity/index.mdx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+<FileTree>
+- src
+  - Transfer.sol
+- script
+  - Transfer.s.sol
+- remappings.txt
+- foundry.toml
+</FileTree>
```
Comment (cor) [inline docs/src/content/docs/integrations/ucs01/solidity/index.mdx:22]: Link to a repo would be good

File: docs/src/content/docs/integrations/ucs01/solidity/index.mdx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    transfer.transferAsset();
+    console.log("complete");
+    vm.stopBroadcast();
+  }
+}
+```
+
+## Funding the Wallet
+
+USDC faucet: [faucet.circle.com](https://faucet.circle.com)
```
Comment (cor) [inline docs/src/content/docs/integrations/ucs01/solidity/index.mdx:111]: the wallet will need SepoliaETH to make the transfer

File: docs/src/content/docs/integrations/ucs01/solidity/index.mdx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+or pick a different token from faucet list here [app.union.build/faucet](https://app.union.build/faucet)
+
+## Executing the Contract
+
+We will transfer 1 USDC from Sepolia to a Union wallet.
+
+The first transfer we will do against a forked network using `anvil`.
+
```
Comment (cor) [inline docs/src/content/docs/integrations/ucs01/solidity/index.mdx:120]: would be good to note which dependencies they need and how to install them

**Review by cor — CHANGES_REQUESTED** [review]

**Comment by PablitoAmaral** [comment]
### My Experience:

At the beginning, everything was very well explained. I only needed to do a quick search in the Foundry Book to find the command to start "forge init," since I had never used it before.

Then, I followed all the steps in the "Funding the Wallet" section, but I realized that with the resources I had, I couldn't do much without a Premium RPC.

I encountered this error: error code -32000: insufficient funds for gas * price + value: balance 100000000000000000, queued cost 96746019474142502, tx cost 23809826780771570, overshot 20555846254914072


I made some unnecessary transactions (rookie mistake), but overall, I think everything was very well explained and easy to follow. Great work!

**Review by cor — APPROVED** [review]

**Pull request merged**